 Project1/style.css

```
@@ -0,0 +1,66 @@
+img {
+    max-width: 100%;
+}
+
+/*comments look like this*/
+h1 {
+    color: hsl(22, 100%, 62%);
+    font-family: Avenir;
+    font-size: 60px;
+    font-style: inherit;
+}
+
+
+
+h2 {
+    color: cadetblue;
+    font-family: Avenir;
+    font-size: 33px;
+    font-weight: 600;
+}
+
+.question-block {
+    color: mediumturquoise;
+    font-family: Avenir;
+    font-style: italic;
+    font-size: 30px;
+    font-weight: 450;
```

Most fonts only accept a `font-weight` of numbering in the 100s, so
```suggestion
    font-weight: 400;
```
or
```suggestion
    font-weight: 500;
```
is more likely to be accepted than `450`